src/main/kotlin/org/malachite/estella/process/domain/RecruitmentProcessDto.kt

```diff
@@ -52,7 +53,7 @@ fun Task.toTaskDto() = TaskDto(
     id,
     this.tests.toBase64String(),
     this.descriptionFileName,
-    this.description.toBase64String(),
+    this.description,
```

Comment:
Tak jak pisałem wyżej - jest to sprawdzone z frontem, czy niczego nie rozwaliłeś tą zmianą?

Reply:
Patrzone przed chwilą z najnowszym mainem, jest git, dobrze robią, pozytywny przekaz leci